server/views/taskList/taskListTemplate.pug

```diff
@@ -26,7 +26,7 @@ block content
       strong.bold The offender is presumed unsuitable for HDC release
   else if user.role === 'CA' && decisions.caRefused
     div.error-banner.marginTopMedium
-      strong.bold Home detention curfew refused by prison case admin#{decisions.refusalReason ?': ' + decisions.refusalReason : ''}
```

Comment:
line 29 with the ternary was a temporary fix in a previous ticket. Ternary no longer needed.

Reply:
you won't be able to remove this ternary - as there will still be lots of data in the database in the bad state. We can only fix it going forward